src/test/regress/sql/multi_utility_statements.sql

```diff
@@ -146,3 +146,7 @@ FETCH ABSOLUTE 5 FROM noHoldCursor;
 FETCH BACKWARD noHoldCursor;
 COMMIT;
 FETCH ABSOLUTE 5 FROM noHoldCursor;
+
+-- Test we don't throw an error for DROP IF EXISTS
+DROP DATABASE IF EXISTS not_existing_database;
+DROP TABLE IF EXISTS not_existing_table;
```

Comment:
We could add an exhaustive list of object types here, but I didn't want to make this PR too hard to review, as an exhaustive list would need the reviewer that we are not missing anything. So that'd be another PR if we need to.